docs/faq.md

```diff
@@ -60,6 +60,39 @@ one megabyte to Flutter apps. `flutter build apk --release` vs.
 we can make this smaller when that becomes a priority. If size is a blocker
 for you, please let us know!
 
+### Does code push work with large applications?
+
+Yes. There is no limit on the size of the application that can be patched with
+code push. As noted [What types of changes does Shorebird code push](below),
```

Comment:
Can we fix the (below) since I don't think the link will work as is?

Reply:
Oh, i see, I swapped them?